src/isIsogram.test.js

```diff
@@ -6,4 +6,22 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for an isogram string', () => {
+    expect(isIsogram('roksa')).toBe(true);
+  });
+
+  it('should return false for non-isogram string', () => {
+    expect(isIsogram('book')).toBe(false);
+    expect(isIsogram('lola')).toBe(false);
+  });
+
+  it('should handle case-insensitivity', () => {
```

Comment:
Similar to line 14, this test case also contains two assertions. Consider splitting this into two separate test cases.